apps/x509.c

```diff
@@ -732,6 +739,12 @@ int x509_main(int argc, char **argv)
         }
     }
 
+    if (bundle && (req || reqfile || newcert || x509toreq || pubkey ||
+                   privkey || clrext || subj || CAfile)) {
+        BIO_printf(bio_err, "Error: -bundle can only be used with other printing options\n");
+        goto err;
+    }
+
```

Comment:
We generally do not enforce such exclusions unless there is important reason to do so.
So better not throw an error in these cases.
Yet it's nice to give a warning, though, in those (part of the) cases where the new option has no effect, saying that the option is ignored.

Reply:
Reduced the exclusions down to just -new and -req.  The presence of either of those will cause a runtime error.

Reply:
The runtime error could easily be avoided for -new and -req, but ok to exclude these because -multi makes little sense with them.

As I wrote, would be nice to warn in cases where -multi has no effect, but I don't insist on this, also because this would be a little cumbersome to find out for which options, and the warning would bring only minor benefit.